app_test_suite/steps/base_test_runner.py

```diff
@@ -59,6 +59,12 @@ def initialize_config(self, config_parser: configargparse.ArgParser) -> None:
             required=True,
             help="Path to the Helm Chart tar.gz file to test.",
         )
+        config_parser.add_argument(
+            "--github-token",
+            env_var="GITHUB_TOKEN",
```

Comment:
I added the env var option but maybe this should be `ATS_GITHUB_TOKEN`?

Reply:
probably a good idea